<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsFormViewEditorClass</class>
 <widget class="QDialog" name="CwmsFormViewEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulareditor für Sichten</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="m_pqlArticleType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(232, 232, 232, 255));</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Formulareditor für Sichten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_pqwData">
      <attribute name="title">
       <string>Allgemeine Daten</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Formulardaten</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="m_pqleName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Druckvorlage</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="m_pqlePrintingTemplate">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="m_pqpbPrintingTemplate">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="wmsgui.qrc">
              <normaloff>:/icons/search24.png</normaloff>:/icons/search24.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Standardkonfiguration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QCheckBox" name="m_pqchbDefault">
            <property name="text">
             <string>Standard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pqchbReadOnly">
            <property name="text">
             <string>Nur lesender Zugriff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pqchbPrint">
            <property name="text">
             <string>Drucken zulassen</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Objektcontainerkonfiguration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QCheckBox" name="m_pqchbSaveAsCsv">
            <property name="text">
             <string>Als CSV speichern zulassen</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pqwViews">
      <attribute name="title">
       <string>Sichten</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QListWidget" name="m_pqlwViews"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_pqpbAddView">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="wmsgui.qrc">
             <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbRemoveView">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="wmsgui.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbViewUp">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="wmsgui.qrc">
             <normaloff>:/icons/arrow-up24.png</normaloff>:/icons/arrow-up24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbViewDown">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="wmsgui.qrc">
             <normaloff>:/icons/arrow-down24.png</normaloff>:/icons/arrow-down24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqcbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbOk">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="wmsgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqcbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormViewEditorClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbOk</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormViewEditorClass</receiver>
   <slot>OKClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbPrintingTemplate</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormViewEditorClass</receiver>
   <slot>SelectPrintingTemplateClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddView</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormViewEditorClass</receiver>
   <slot>AddViewClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveView</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormViewEditorClass</receiver>
   <slot>RemoveViewClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbViewUp</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormViewEditorClass</receiver>
   <slot>ViewUpClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbViewDown</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormViewEditorClass</receiver>
   <slot>ViewDownClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OKClickedSlot()</slot>
  <slot>SelectPrintingTemplateClickedSlot()</slot>
  <slot>AddViewClickedSlot()</slot>
  <slot>RemoveViewClickedSlot()</slot>
  <slot>ViewUpClickedSlot()</slot>
  <slot>ViewDownClickedSlot()</slot>
 </slots>
</ui>
